### PR TITLE
Improve server-side rendering documentation

### LINKED ISSUE #2168: mhchem for server-side Katex
**Is your feature request related to a problem? Please describe.**
As far as I'm aware, there's no way to enable Katex's mhchem implementation when rendering server-side (in NodeJS). Some of us want to use this extension outside of the browser.

**Describe the solution you'd like:**
I'm not sure what the appropriate solution is. It sounds like what needs to happen is...
1. skip __defineMacro() invocations if running outside of a browser.
2. provide a streamlined way of injecting the relevant macros into KatexOptions when invoking renderToString().

```js
const out = katex.renderToString('...', {
    macros: {
        "\\ce": function(context) {
            return chemParse(context.consumeArgs(1)[0], "ce")
          },
        "\\pu": function(context) {
            return chemParse(context.consumeArgs(1)[0], "pu");
          },
        "\\tripledash": "{\\vphantom{-}\\raisebox{2.56mu}{$\\mkern2mu"
          + "\\tiny\\text{-}\\mkern1mu\\text{-}\\mkern1mu\\text{-}\\mkern2mu$}}"
    }
})
```

**Link to or name of a (La)TeX package that provides the same feature:**
N/A

**Describe alternatives you've considered:**
So far all I've tried is forking mhchem.js. 

**Additional context:**
N/A

### LINKED ISSUE #2176: Server-side rendering guide needs more clarification
Hi, I have checked docs several times, but still can't figure out next questions: 

1. Do I need to include `fonts` and `katex.js` files if I am doing server side rendering (i.e. `renderToString`)? I am including `katex.min.css` file and it looks fine for simple examples, however not sure if it's enough for complex math stuff.
1. Is it possible to completely inline styles while rendering server side (like [shiki](https://github.com/octref/shiki) does)?

Thanks in advance for the answer.

### PR DESCRIPTION
**Pull request by edemaine**
* Fix #2176: mention what you need to include when server-side rendering.
* Fix #2168: mention how to use mhchem on Node.

### CODE REVIEW

File: docs/node.md [hunk 1]
```diff
@@ -75,4 +75,13 @@ import katex from 'katex/dist/katex.mjs'
...
 > The ES module contains ES6 syntaxes and features, and may need transpiling to
-use in old environments:
+use in old environments.
+
+See the [API](api.html) for available functions in the `katex` module.
+
+## Including in webpages
+
+To render any HTML generated by e.g. `katex.renderToString`, you will need
+still need to link the CSS file, make the KaTeX font files available to the
```
Comment (ipelekhan) [inline docs/node.md]: > you will need still need

Seems like a typo.

Comment (edemaine) [reply docs/node.md]: Thanks for spotting! Fixed.

**Comment by edemaine** [comment]
I added another paragraph that documents how to use mhchem in Node. This is ready for review.

File: docs/node.md [hunk 2]
```diff
@@ -75,4 +75,25 @@ import katex from 'katex/dist/katex.mjs'
...
+
+## Using mhchem extension
+
+The [mhchem extension](https://github.com/Khan/KaTeX/tree/master/contrib/mhchem)
+adds its features by modifying the `katex` module.  So you can use KaTeX with
+mhchem in Node as follows:
+
+```js
+require('katex/dist/contrib/mhchem.js'); // modify katex module
+const katex = require('katex');
```
Comment (ylemkimon) [inline docs/node.md]: I think importing the extension after importing KaTeX is more intuitive.

Comment (edemaine) [reply docs/node.md]: Sounds good. (I tested that it works both ways.) Changed!

**Pull request merged**